k8s/helmfile/env/production/tool-quickstatements.values.yaml.gotmpl

```diff
@@ -1,7 +1,7 @@
 replicaCount: 1
 
 platform:
-  mediawikiBackendHost: mediawiki-138-app-backend.default.svc.cluster.local
+  mediawikiBackendHost: mediawiki-139-app-backend.default.svc.cluster.local
```

Comment:
As noted above, there is more of a chance of things being broken for sites if pointing to new code while old DB schemas are used.
Though the "real" fix for the oauth tools lies in having some service infront of all mediawiki services directing things to the correct versions, and doing that autmatically.